.github/workflows/ci_cd.yml

```
@@ -102,6 +96,26 @@ jobs:
           just precommit
           just lint
 
+  add-stack-label:
+    name: Add stack label
+    runs-on: ubuntu-latest
+    needs:
+      - get-changes
+
+    steps:
+      - uses: actions/checkout@v3
```

nit, sorry: Jobs should have names.

```suggestion
      - Checkout repository
        uses: actions/checkout@v3

```